packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -924,6 +947,11 @@ export enum ComputeType {
   LARGE = 'BUILD_GENERAL1_LARGE'
 }
 
+export enum ImagePullCredentialsType {
```

Comment:
Some docs here explaining what is the difference between the 2 options would be very useful.